File: jscs_array/taskC.js
let arr = [ 1, 2, -5, 8, -3, 9 , -7 ];
let limit = 4;
let arr1 = [];
for(let i in arr) {
    if(arr[i] > 4) {
        arr1.push(arr[i]);
    }
}
console.log(arr1);

File: jscs_array/taskC_cs/taskC.cs
int[] arr = { 1, 2, -5, 8, -3, 9 , -7 };
int limit = 4;
int[] arr1 = {1};
for(int i = 0; i < arr.Length; i++) {
    if(arr[i] > limit) {
        Console.WriteLine(arr[i]);
    }
}
